### ISSUE TITLE
fix homepage WCAG compliance

### ISSUE THREAD

**Issue by nikolaitandberg** [nikolaitandberg/idatt2106-2025-09-frontend#144]
Not logged in user: 
- missing text-label for login-page link
- bad contrast for text-leaf
- alert not focusable
